domains/transfer-agreements/TransferAgreements.API/API.IntegrationTests/Controllers/TransferAgreementsControllerTest.cs

```diff
@@ -178,39 +177,35 @@ await SeedData(new List<TransferAgreement>()
 
         var getTransferAgreement = JsonConvert.DeserializeObject<TransferAgreementDto>(await get.Content.ReadAsStringAsync());
 
-        getTransferAgreement.Should().NotBeNull();
+        var settings = new VerifySettings();
+        settings.ScrubMembersWithType(typeof(long));
```

Comment:
If you used fixed Start- and EndDates in stead of DateTimeOffset.UtcNow, then it is not need to scrub long-values